### PR TITLE
fix a couple bugs, cleanup, and new property

### PR DESCRIPTION
**Pull request by robertleeplummerjr**
Fixed out of need.  Thanks for this lib.

### CODE REVIEW

**Comment by robertleeplummerjr** [comment]
/me hopes these changes can be implemented today if they meet your requirements...

File: leaflet.measurecontrol.js [hunk 1]
```diff
@@ -93,17 +102,20 @@ L.Control.MeasureControl = L.Control.extend({
...
+            className = 'leaflet-control-draw';
 
         this._container = L.DomUtil.create('div', 'leaflet-bar');
 
         this.handler = new L.Polyline.Measure(map, this.options.handler);
 
         this.handler.on('enabled', function () {
+            self.enabled = true;
             L.DomUtil.addClass(this._container, 'enabled');
         }, this);
```
Comment (leplatrem) [inline leaflet.measurecontrol.js]: nitpick: the `on()` is passed a reference to `this`, the `self` trick isn't necessary

**Comment by leplatrem** [comment]
It looks good to me, except the superfluous `self`

**Comment by robertleeplummerjr** [comment]
de-superfluous-ness-ivity-ism-ed

:P
Thanks for the review!  Wish granted.

**Comment by robertleeplummerjr** [comment]
ping

**Comment by fredericbonifas** [comment]
Thanks @robertleeplummerjr !

**Comment by robertleeplummerjr** [comment]
open source rocks.

**Pull request merged**